package.json

```diff
@@ -25,7 +25,8 @@
     "devenv:stop": "docker-compose -f docker-compose.dev.postgres.yml -f docker-compose.dev.stacks-blockchain.yml -f docker-compose.dev.bitcoind.yml down -v -t 0",
     "devenv:logs": "docker-compose -f docker-compose.dev.postgres.yml -f docker-compose.dev.stacks-blockchain.yml -f docker-compose.dev.bitcoind.yml logs -t -f",
     "docs:generate": "cd docs && npm run generate:types && npm run generate:docs && cd ../client && npm run generate:docs",
-    "docs:deploy": "npm run docs:generate && cd docs && gulp deployDocs"
+    "docs:deploy": "npm run docs:generate && cd docs && gulp deployDocs",
+    "test:bns": "npm run devenv:deploy -- -d && cross-env NODE_ENV=development jest --config ./jest.config.bns.js --coverage --runInBand"
```

Comment:
I think this needs to include `npm run devenv:stop` at the end like this:

`npm run devenv:deploy -- -d && cross-env NODE_ENV=development jest --config ./jest.config.bns.js --coverage --runInBand; npm run devenv:stop`


Otherwise the background docker processes won't exit after test finishes.